android/src/com/frostwire/android/gui/fragments/BrowsePeerFragment.java

```diff
@@ -585,6 +628,178 @@ public void onCheckedChanged(CompoundButton buttonView, boolean isChecked) {
         return selectAllCheckboxListener;
     }
 
+    private class MyFilesActionModeCallback implements android.support.v7.view.ActionMode.Callback {
+        private ActionMode mode;
+        private Menu menu;
+        private int numChecked;
+
+        public void onItemChecked(Context context, int numChecked) {
+            this.numChecked = numChecked;
+            if (mode != null) {
+                mode.setTitle(numChecked + " " + context.getString(R.string.selected));
+                mode.invalidate();
+            }
+        }
+
+        @Override
+        public boolean onCreateActionMode(ActionMode mode, Menu menu) {
+            this.mode = mode;
```

Comment:
not a big fan of keeping this references (I understand the need), but I can't propose anything better